README.md

```diff
@@ -33,19 +32,30 @@ Find out more about Managed Components [here](https://blog.cloudflare.com/zaraz-
 
 > Fields are properties that can/must be sent with certain events
 
-### Tiktok Event Name `string` _required_
+### Tiktok Standard Event Name `string` _required_
 
 `ev` can be \"ContentView\", \"PlaceAnOrder\", \"Add to Cart\" one of [the standard Tiktok events](https://ads.tiktok.com/marketing_api/docs?id=1727541103358977).
 
+### Custom Event Name `string` _required_
+
+`cev` can be any custom event name. Do not use sensitive words in the name of Custom Events. [Learn more](https://business-api.tiktok.com/portal/docs?id=1771101303285761).
 
 ### Email `string`
 
 `email` user email. Automatically hashed.
 
-### Phone Number `string`
+### Phone `string`
+
+`phone` The phone number of the customer. Normalize the phone number to E.164 format (for example, '+12133734253'). Country code must be included and prefixed with the '+' sign, without any parentheses or leading '0's. No spaces, letters or symbols except the '+' for the country code are included. Zaraz will SHA-256 hash the phone number automatically. [Learn more](https://business-api.tiktok.com/portal/docs?rid=5ipocbxyw8v&id=1771100865818625).
```

Comment:
we have to provide backward compatibility because if we release it, people with previous tik tok configs will not get the phone number for example. Can we make sure phone and phone_number are both used in the MC?